webpack.config.js

```diff
@@ -12,6 +13,7 @@ const FixStyleOnlyEntriesPlugin = require('webpack-fix-style-only-entries');
 
 const {env} = process;
 
+const PLATFORMS_URL = 'https://docs.sentry.io/_platforms/_index.json';
```

Comment:
I wouldn't mind pulling this stuff out into a separate file actually. Not sure where exactly though

Reply:
I also wanna separate these but I think we can delay this until we actually have a separate FE repo if we can as creating a new `build` folder only for a few functions feels like overkill to me.